workers/importscripts_mime_local.worker.js

```diff
@@ -0,0 +1,68 @@
+// META: global=worker
+//
+// Tests for https://github.com/whatwg/html/issues/8869
+// Targeting Workers only because ServiceWorkers don't
+// have access to URL.createObjectURL
+
+importScripts("/resources/testharness.js");
+
+let test_cases = [
+  // Supported mimetypes:
+  ["text/javascript", true],
+  ["application/javascript", true],
+  ["text/ecmascript", true],
+
+  // Blocked mimetpyes:
+  ["image/png", false],
+  ["text/csv", false],
+  ["video/mpeg", false],
+
+  // Legacy mimetypes:
+  ["text/html", false],
+  ["text/plain", false],
+  ["application/xml", false],
+  ["application/octet-stream", false],
+
+  // Potato mimetypes:
+  ["text/potato", false],
+  ["potato/text", false],
+  ["aaa/aaa", false],
+  ["zzz/zzz", false],
+
+  // Parameterized mime types:
+  ["text/javascript; charset=utf-8", true],
+  ["text/javascript;charset=utf-8", true],
+  ["text/javascript;bla;bla", true],
+  ["text/csv; charset=utf-8", false],
+  ["text/csv;charset=utf-8", false],
+  ["text/csv;bla;bla", false],
+
+  // Funky capitalization:
+  ["Text/html", false],
+  ["text/Html", false],
+  ["TeXt/HtMl", false],
+  ["TEXT/HTML", false],
+];
+
+for (var test_case of test_cases) {
```

Comment:
I think this could be a bit clearer:
```suggestion
for (const ([mimeType, isScriptType]) of test_cases) {
```
(Generally we'd also use objects for this reason to express the individual test cases, but I think that's fine here.)

Reply:
I felt the same when authoring.  
Do you want me to also update [importscripts_mime.any.js](https://github.com/web-platform-tests/wpt/blob/master/workers/importscripts_mime.any.js)? Maybe in another PR would be preferable, or here is fine?

Reply:
Yeah feel free to update it as part of this PR.

chrome-dev-stability is known to have timeout issues. If it happens again when you next push we'll ask for admin merge.